File: Estrutura_Sequencial/JavaScript/Ex1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

funcionario = parseInt(prompt());
horas = parseInt(prompt());
salario = parseFloat(prompt());

salariof = salario * horas;
console.log("NUMBER = " + funcionario +
            "\nSALARY = U$ " + salariof.toFixed(2))

File: Estrutura_Sequencial/C#/Ex1008.cs
using System;

class MainClass {
    public static void Main (string[] args) {

        int funcionario = int.Parse(Console.ReadLine());
        int horas = int.Parse(Console.ReadLine());
        float salario = float.Parse(Console.ReadLine());

        float salariof = salario * horas;
        Console.WriteLine($"NUMBER = {funcionario}\nSALARY = U$ {salariof:F2}");
        
    }
}